Documentation/cmdref/cilium-bugtool_completion_zsh.md

```diff
@@ -0,0 +1,40 @@
+<!-- This file was autogenerated via cilium cmdref, do not edit manually-->
+
+## cilium-bugtool completion zsh
+
+generate the autocompletion script for zsh
+
+### Synopsis
+
+
+Generate the autocompletion script for the zsh shell.
+
+If shell completion is not already enabled in your environment you will need
+to enable it.  You can execute the following once:
+
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+To load completions for every new session, execute once:
+# Linux:
+$ cilium-bugtool completion zsh > "${fpath[1]}/_cilium-bugtool"
+# macOS:
```

Comment:
On a quick look at the new documentation files, it looks like Markdown for the description of the completion commands will be broken.
I understand these are generated files, do you know if we have a way to format or include this text differently somehow?

Reply:
I don't know, will need to look into this, unless someone else already knows what to do?